### ISSUE TITLE
add support for D455 camera

### ISSUE THREAD

**Issue by ste93** [robotology/yarp-device-realsense2#11]
until now the d455 camera is not working with this device, the support must be added but checking which distortion is returning (maybe adding the INVERSE_BROWN_CONRADY distortion) and must be checked if is useful to have those parameters, together with the check if the image returned is distorted or not

**Comment by ste93** [comment]
I've checked with realsense developers https://github.com/IntelRealSense/librealsense/issues/8810#issuecomment-818312496 and D455 camera sends distorted images, so I think we have to implement inverse brown conrady distortion to use this camera. The distortion is not so high so it can also be used undistorted, but the correct way to add it is to implement inverse brown conrady distortion.

**Comment by traversaro** [comment]
Thanks @ste93 for looking into that. From this comment, it seems that indeed there is a bug in how the D455 camera is handled in https://github.com/IntelRealSense/realsense-ros .

**Comment by ste93** [comment]
yes, I think that ros has a bug inside (that I think they're trying to solve) since it returns the wrong distortion model. For the realsense driver I think that there are also there some bugs (as can be seen here https://github.com/IntelRealSense/librealsense/issues/7335) the last commit is 2 days ago and it referenced D455 and inverse brown conrady, so I think this is still a work in progress.

**Comment by ste93** [comment]
I think this can be useful for the addition of inverse brown conrady https://github.com/IntelRealSense/librealsense/commit/90fa2ed4bf9ff845c8a7d8bd78a7552f9b81dd43

**Comment by ste93** [comment]
just for tracking, new librealsense has been released yesterday and it seems that they have added undistortion also with inverse brown conrady, but camera firmware must be updated. We will check it when R1SN003 will return from Turin.
